### PR TITLE
Fix Typo in Events Code of Conduct

### PR DESCRIPTION
**Pull request by mbssaiakhil**

### CODE REVIEW

**Comment by mrunalp** [comment]
Could you sign your commit? `git commit --signoff`

**Comment by mbssaiakhil** [comment]
Thanks @mrunalp, I have signed my commit. Kindly review and merge.

File: code-of-conduct.md [hunk 1]
```diff
@@ -52,4 +52,4 @@ The Kubernetes team does not condone any statements by speakers contrary to thes
 team reserves the right to deny entrance and/or eject from an event (without refund) any individual found to
 be engaging in discriminatory or offensive speech or actions.
 
-Please bring any concerns to to the immediate attention of Kubernetes event staff
+Please bring any concerns to the immediate attention of Kubernetes event staff
```
Comment (TomSweeneyRedHat) [inline code-of-conduct.md]: Could you touch this up a bit more?  Perhaps to:

Please bring any concerns to the immediate attention of the Kubernetes event staff.

Thanks!

**Comment by rhatdan** [comment]
LGTM
@runcom Please merge, no need to wait for/run tests.

File: code-of-conduct.md (hunk 1 above)
Comment (mbssaiakhil) [inline code-of-conduct.md]: Thanks @TomSweeneyRedHat, have amended my changes with your suggestion. Kindly review and merge.

**Comment by rhatdan** [comment]
@mrunalp Please merge, no need to wait for/run tests.

**Pull request merged**